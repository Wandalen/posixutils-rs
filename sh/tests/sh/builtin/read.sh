echo test > $TEST_WRITE_DIR/read.txt
read var < $TEST_WRITE_DIR/read.txt
echo $?
echo $var

echo 'val1 val2' > $TEST_WRITE_DIR/read.txt
read var1 var2 < $TEST_WRITE_DIR/read.txt
echo $?
echo $var1
echo $var2

echo 'val1 val2 val3 val4' > $TEST_WRITE_DIR/read.txt
read var1 var2 var3 < $TEST_WRITE_DIR/read.txt
echo $?
echo $var1
echo $var2
echo $var3

echo val1 > $TEST_WRITE_DIR/read.txt
read var1 var2 < $TEST_WRITE_DIR/read.txt
echo $?
echo $var1
echo $var2

rm $TEST_WRITE_DIR/read.txt