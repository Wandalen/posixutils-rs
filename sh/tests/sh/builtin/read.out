0
test
0
val1
val2
0
val1
val2
val3 val4
0
val1

